packages/react-reconciler/src/ReactFiberReconciler.new.js

```diff
@@ -307,7 +310,14 @@ export function createHydrationContainer(
   // enqueue the callback if one is provided).
   const current = root.current;
   const eventTime = requestEventTime();
-  const lane = requestUpdateLane(current);
+  let lane = requestUpdateLane(current);
+
+  // TODO: Do we want container updates to be in a rAF?
+  // If not, what to do about tests/act?
+  // TODO: Is there a better way to do this?
+  if ((lane & UnknownLane) !== NoLane) {
+    lane = DefaultLane;
+  }
```

Comment:
What do we want to do for updates inside `render` or `hydrateRoot`? I'm forcing those to `Default` if there's no other priority (mostly because it it will fail a ton of tests otherwise). If we want to do that, is this the best way to do it?